### PR TITLE
Reset pathbar on lost focus

### LINKED ISSUE #2068: Path bar text seems to overlap the path bar drawing.
### What Happened?

Please, see below.

https://user-images.githubusercontent.com/38897885/184465283-960b5375-c6ba-41f7-a37a-6fa7e686763e.mp4

### Steps to Reproduce

1. Double-click on the path bar.
2. Minimize and restore the window.

### Expected Behavior

I'm not sure, I think just the path bar draw should appear

### OS Version

6.x (Odin)

### Software Version

Latest release (I have run all updates)

### Log Output

_No response_

### Hardware Info

_No response_

### PR DESCRIPTION
**Pull request by darthgelum**
Fixes #2068
Based on solution by @jeremypw #2069 but checks `Gdk.WindowState.Focus`

Works on hiding by button and clicking icon

### CODE REVIEW

**Comment by jeremypw** [comment]
@darthgelum Would you update this PR please?  Thanks.

**Comment by darthgelum** [comment]
Sorry, will update in a  week. Thanks!

File: libcore/Widgets/BasicBreadcrumbsEntry.vala [hunk 1]
```diff
@@ -335,7 +335,12 @@ namespace Files.View.Chrome {
                 }
 
                 // Do not lose entry text if another window is focused
-                if (((Gtk.Window)(get_toplevel ())).has_toplevel_focus) {
+                // Reset if window has top level focus or is iconified
+                var top_level = (Gtk.Window)(get_toplevel ());
+                if (top_level.has_toplevel_focus ||
+                     (top_level.get_window ().get_state () & Gdk.WindowState.FOCUSED) > 0) ||
```
Comment (jeremypw) [inline libcore/Widgets/BasicBreadcrumbsEntry.vala:341]: There are unmatched brackets here so this does not compile.

**Review by jeremypw — CHANGES_REQUESTED** [review]

**Comment by jeremypw** [comment]
When the bracket issue is fixed this fixes the linked issue and no regressions were found.

**Comment by jeremypw** [comment]
To retarget on Gtk4 version

**Pull request closed without merging**